js/activity_list.js

```diff
@@ -62,10 +62,14 @@ const activityListAction = function (initializer) {
             const entry = $("<div>").data("appInfo", l).appendTo(container).click(startHView).addClass("entry");
 
             if (list.hasIcons) {
```

Comment:
if we are always showing icons, can we get rid of this check?
Which case doesn't has icons?

Reply:
Sounds good to me!